app/frontend/packs/app-back-link.js

```diff
@@ -4,7 +4,8 @@ const initBackLinks = () => {
   const backLinks = document.querySelectorAll('.app-back-link--fallback')
```

Comment:
We currently only `initBackLinks` in the candidate js file. Can it be moved to the main js file?

Reply:
Yes, we're doing this